src/app/child-dev-project/attendance/add-day-attendance/add-day-attendance.component.ts

```diff
@@ -38,8 +36,8 @@ export interface AddDayAttendanceConfig {
   ],
   standalone: true,
 })
-export class AddDayAttendanceComponent {
-  config?: AddDayAttendanceConfig;
+export class AddDayAttendanceComponent implements AddDayAttendanceConfig {
+  @Input() sortParticipantsBy: any;
```

Comment:
```suggestion
  @Input() sortParticipantsBy: string;
```
Also the interface is not really useful as it only provides this field which is overwritten here and its only used once.